index.html

```diff
@@ -0,0 +1,104 @@
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>End of the World Event</title>
+    <link rel="icon" type="image/x-icon" href="ressources/event-image/EoW-red-circle.png">
+
+  </head>
+  <body>
+    <header class="header">
+      <section class="black-banner-container">
+        <div class="black-banner-items">
+          <a href="https://facebook.com/microverse" target="_blank" rel="noopener noreferrer" class="fa fa-facebook"></a>
+          <a href="https://twitter.com/microverse" target="_blank" rel="noopener noreferrer" class="fa fa-twitter"></a>
+          <a href="#English" target="_blank" rel="noopener">English</a>
+          <a href="#MyPage" target="_blank" rel="noopener">My Page</a>
+          <a href="#Logout" target="_blank" rel="noopener">Logout</a>
+        </div>
+      </section>
+      <section class="white-banner">
+        <div class="logo-container">
+          <a href="/" class="logo-container-2">
+            <img src="ressources/event-image/EoW-Logo.png" alt="A red logo for the End of the World Event" class="logo">
+          </a>
+        </div>
+        <div class="menu-container">
+          <div id="overlay-screen" class="overlay">
+            <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+            <nav class="overlay-content menu">
+              <a href="./about.html">About</a>
+              <a href="#Program">Program</a>
+              <a href="#Jobs">Jobs</a>
+              <a href="#Sponsor">Sponsor</a>
+              <a href="#News">News</a>
+            </nav>
+          </div>
+          <span class="hamburger" onclick="openNav()">&#9776;</span>
+        </div>
+        <nav class="nav-menu">
+          <a href="./about.html">About</a>
+          <a href="#Program">Program</a>
+          <a href="#Jobs">Jobs</a>
+          <a href="#Sponsor">Sponsor</a>
+          <a href="#News">News</a>
+        </nav>
+        <div class="button-container">
+          <button class="header-button" type="button" name="button">CC Campaing</button>
+        </div>
+      </section>
+    </header>
+    <section class="welcome-screen">
+      <div class="white-overlay">
+        <div class="welcome-text">
+          <p class="welcome-user">Hello Beautiful People</p>
+          <h2 class="page-title">End of the World Event 2022</h2>
+          <p class="event-intro">The end is near, and the world is doing one last event to finish lives of 7 billions people this year. for the occasion amazing VIP will accompagny you along the way to leave all together. Would you rather be alone and bored, or dance like there is no tomorrow and have no regrets ?</p>
+          <h3>2022.02.22(TUE)</h3>
+          <h4>@Camp Nou, Barcelona, Spain</h4>
+        </div>
+      </div>
+    </section>
+    <section class="main-program">
```

Comment:
- [ ] In this section please adjust the colors(background color), font of the program cards based on the given design

Yours:
![Screenshot from 2021-11-16 22-44-17](https://user-images.githubusercontent.com/62705947/142071107-681d8756-d771-499f-8fa0-69dc014b2dce.png)
Given design:
![Screenshot from 2021-11-16 22-44-20](https://user-images.githubusercontent.com/62705947/142071125-5c1a348e-8748-4d47-b3f3-fcb8d0b34819.png)

Reply:
- [ ] Please update your mobile version at this section in order to match the original:

Original:
![Screenshot from 2021-11-16 22-48-28](https://user-images.githubusercontent.com/62705947/142071419-9b050c7c-50c4-4dc7-9ceb-f5de473cc820.png)
Yours:
![Screenshot from 2021-11-16 22-49-34](https://user-images.githubusercontent.com/62705947/142071492-0bc4154b-ceb6-4427-ad58-630d7d925b98.png)

Reply:
I didn't see the design from the mobile version, thanks for pointing it out

Reply:
Well done here, however, the title needs to be visible, please change the color of the title.

![html](https://user-images.githubusercontent.com/55635977/142187673-62c13642-7a39-4d3b-82a0-f04f84abf436.png)

Reply:
I don't know why the image background didn't load on your computer, I just checked and there is an image on the background - please check again thanks